domain/src/main/kotlin/no/nav/su/se/bakover/domain/søknadsbehandling/Søknadsbehandling.kt

```diff
@@ -995,3 +994,19 @@ sealed class Søknadsbehandling {
         }
     }
 }
+
+// TODO jah, jm: Fjern på sikt?
```

Comment:
TODO; Trenger en løsning for frontend for å evt kunne fjerne.

Reply:
Brukes denne av databaselaget nå? Den brukes vel uansett av andre eksterne lag som pdf, journalføring,statistikk,vedtak?

Reply:
Denne kommer vi ikke til å løse i denne PR.